vue-frontend/src/components/portfolio/VideoSection.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="tw-mt-12">
+    <div class="tw-container tw-my-36">
+      <div class="tw-flex tw-flex-row">
+        <div class="tw-basis-1/4">
+          <p class="v2-normal-text tw-font-bold tw-w-11/12">The Challenge</p>
+        </div>
+        <div class="tw-flex-1 w-64 ...">
+          <p class="v2-normal-text">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard. Lorem Ipsum
+            is simply dummy text of the printing and typesetting industry. Lorem
+            Ipsum is simply dummy text of the printing and typesetting industry.
+            Lorem Ipsum has been the industry's standard. Lorem Ipsum is simply
+            dummy text of the printing and typesetting industry.
+          </p>
+          <ul class="tw-list-none tw-inline-flex tw-space-x-9 tw-mt-12">
+            <li>
+              <a class="v2-normal-3-text tw-font-semibold" :href="careerURL"
+                ><span class="underline-text">Laun</span>ch Project</a
+              >
+            </li>
+            <li>
+              <a
+                class="v2-normal-3-text tw-font-semibold"
+                :href="blogsURL"
+                target="_blank"
+                >App Store</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div
+        class="tw-bg-[#AFADAD] ... tw-box-border tw-w-10/11 tw-p-72 tw-my-32 tw-border-4 ..."
```

Comment:
I am not understanding this class. What it will do? and for color, we can add it in tailwind.config.js and use here.

Reply:
this is simple box in which later we will add video .